File: dp_labs_and_solutions/original/dp_labs_and_solutions/dp_java_labs_and_solutions/clone/lab/Lettuce.java
// Lettuce.java

public abstract class Lettuce
{
	public Lettuce(double aPrice)
	{
		myPrice = aPrice;
	}

	public Lettuce(Lettuce aLettuce)
	{
		myPrice = aLettuce.myPrice;
	}

	private double myPrice;
}

File: dp_labs_and_solutions/original/dp_labs_and_solutions/dp_cpp_labs_and_solutions/abstract_factory/lab/lettuce.cpp
// lettuce.cpp

#include "lettuce.h"

lettuce::lettuce(double a_price)
{
	my_price = a_price;
}

lettuce::lettuce(const lettuce & other)
{
	my_price = other.my_price;
}

lettuce::~lettuce()
{
}
